lib/ArduinoNative/src/WString.h

```diff
@@ -478,6 +499,103 @@ class String
         return isEmptyFlag;
     }
 
+    /**
+     * If the find pattern is found in the String, it is replaced by the String replace.
+     *
+     * @param[in] find      Pattern
+     * @param[in] replace   Replacement for Pattern
+     */
+    void replace(const String& find, const String& replace)
+    {
+        unsigned int replaceLen = replace.length();
+        unsigned int findLen    = find.length();
+
+        if ((0 == length()) || (0 == findLen))

```

Comment:
m_buffer can be a nullptr

Reply:
Resolved with [0a851f2](https://github.com/BlueAndi/DroidControlShip/pull/30/commits/0a851f278e883ffcb55ac8a7f9862882d4d7a118)